src/main/java/org/usfirst/frc/team5507/robot/commands/PivotHigh.java

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team5507.robot.commands;
+
+import org.usfirst.frc.team5507.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class PivotHigh extends Command {
+  public PivotHigh() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    requires(Robot.m_pivot);
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+    Robot.m_pivot.high();
+  }
+
+  // Make this return true when this Command no longer needs to run execute()
+  @Override
```

Comment:
Should the `isFinished` method return true at some point?  If the pivot takes 140 milliseconds to run, should it return false until 140 ms have passed?
Otherwise, how does the system know when the command is complete?